layout-server/src/main/java/org/finos/vuu/layoutserver/model/ApplicationLayout.java

```diff
@@ -0,0 +1,25 @@
+package org.finos.vuu.layoutserver.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.finos.vuu.layoutserver.utils.JsonNodeConverter;
+
+import javax.persistence.Column;
+import javax.persistence.Convert;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Data
+@Entity
+@RequiredArgsConstructor
+@AllArgsConstructor
+public class ApplicationLayout {
+    @Id
+    private String username;
+
+    @Convert(converter = JsonNodeConverter.class)
```

Comment:
It throws a 400. Remember that the definition is now embedded as fully-formed JSON within the request body (see Postman screenshot in PR description), so this would be handled by Spring in exactly the same way as any other malformed JSON body.